WebApp-OpenIDConnect-DotNet/OpenIdConnectOptionsSetup.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+
+namespace WebApp_OpenIDConnect_DotNet
+{
+    public class OpenIdConnectOptionsSetup : IConfigureOptions<OpenIdConnectOptions>
+    {
+        public OpenIdConnectOptionsSetup(IOptions<AzureAdB2COptions> b2cOptions)
+        {
+            AzureAdB2COptions = b2cOptions.Value;
+        }
+
+        public AzureAdB2COptions AzureAdB2COptions { get; set; }
+
+        public void Configure(OpenIdConnectOptions options)
+        {
+            options.ClientId = AzureAdB2COptions.ClientId;
+            options.Authority = AzureAdB2COptions.Authority;
+
+            // TODO: Is this needed? Is the default sufficient?
```

Comment:
We need to confirm. In the past, the default was the full URL with the xml namespace.
If that has changed, the we can drop this.